src/resolvers/admin.ts

```diff
@@ -23,4 +24,20 @@ export class AdminResolver {
     await killSwitch.save()
     return { isKilled }
   }
+
+  @Mutation(() => GeneratedSignature)
+  async generateSignature(@Arg('bytes', () => String) bytes: string) {
+    if (!config.appPrivateKey) {
+      throw new Error('No PRIVATE_KEY provided to generate signature')
+    }
+
+    if (!cryptoIsReady()) {
+      throw new Error('@polkadot/util-crypto is not ready')
+    }
+
+    const keypair = sr25519PairFromSeed(config.appPrivateKey)
```

Comment:
`sr25519PairFromSeed` function suggests it actually accepts a seed (so I think mnemonic?) and not a private key? Is that the case?

Reply:
The result is signature, so let's name the variable that. `ket` can be misleading